build.sbt

```diff
@@ -85,6 +85,7 @@ credentials ++= {
     keyId <- sys.env.get("SIGN_KEY_ID")
   } yield Credentials("GnuPG Key ID", "gpg", keyId, "ignored")
 }
+publishM2Configuration := publishM2Configuration.value.withOverwrite(true) // allows overwriting local .m2 publishings
```

Comment:
are we sure this is only for local `.m2`  and not on maven central?

Reply:
The description for `publishM2Configuration` is `"Configuration for publishing to the local Maven repository."`, so I believe this should be only for local `.m2` repo.